libs/langchain/langchain/schema/vectorstore.py

```diff
@@ -608,7 +608,7 @@ def as_retriever(self, **kwargs: Any) -> VectorStoreRetriever:
         """
         tags = kwargs.pop("tags", None) or []
         tags.extend(self._get_retriever_tags())
-
+        
```

Comment:
```suggestion

```

Reply:
Yes, updated my ruff version and removed that extra space line.

Reply:
1 more formatting error I think - just a newline where those spaces were removed

Reply:
If you run `make format` from libs/langchain, this should handle all these things automatically!

Reply:
Thanks. Fixed. (sorry was running "lint" from antoher folder and it prints all kinds of other unrelated things. Will know about make format going fwd)

Reply:
No problem! Thanks for fixing.